Sources/APNS/Application+APNS.swift

```diff
@@ -68,7 +68,8 @@ extension Application.APNS: APNSwiftClient {
         priority: Int?,
         collapseIdentifier: String?,
         topic: String?,
-        logger: Logger?
+        logger: Logger?,
+        apnsID: UUID?
```

Comment:
This is a breaking change. To avoid this, add a default value (` = nil`) to the `apnsID` parameter.

Reply:
Thanks I have fix it.